0110    javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,11 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+
+  for (let i = 0; i < words.length; i++) {
+    words[i] = words[i] + suffix;
+  }
+
```

Comment:
word of caution with this, the way you have this written will _mutate_ the input array. It would be better to declare a local variable like `result = []` and push the concatenated words to the `result` array and return that instead.